desktop-src/medfound/topology-attributes.md

```diff
@@ -14,7 +14,8 @@ The following attributes apply to topologies.
 
 | Attribute                                                                                                | Description                                                                                                                                                                                                                        |
 |----------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [MF\_TOPOLOGY\_DXVA\_MODE](mf-topology-dxva-mode.md)                                                    | Specifies whether the topology loader enables Microsoft DirectX Video Acceleration (DXVA) in the topology.                                                                                                                         |
+| [MF\_TOPOLOGY\_DXVA\_MODE](mf-topology-dxva-mode.md)                                                    | Specifies whether the topology loader enables Microsoft DirectX Video Acceleration (DXVA) in the topology.
+| [M\_TOPOLOGY\_ENABLE\_XVP\_FOR_PLAYBACK]()                                                              |.
```

Comment:
Please, create a file for the attribute in the medfound (.md) and remember to describe the attribute because I lack information.